index.html

```diff
@@ -0,0 +1,207 @@
+                            </ul>
+                     </nav>
+              </div>
+          </div>
+ </header>
+
+ <div class="welcom">
+  <h1 class="welcom-h1">Hello</h1>
+
+  <p class="welcom-p1">
+       Revolutionizing Education Assessment: The Integration of Artificial Intelligence for Exam Correction
+  </p>
+  <p class="welcom-p2">
+       By using artificial intelligence in the improvement of education, we will succeed in the success of well-qualified students and we will also boost technology and other fields, so it is a closed loop that will always evolve in time and in space.
+  </p>
+  <p class="welcom-p3">
+     <span> 2023/2/2 to 4  (thusday)</span> <br>
+      AT statup ZoumTech .
+  </p>
+
+ </div>
+
+       
+ <div class="main-project">
+       <div class="title-Main">Main Program</div>
+       <div class="dashProjectMain"></div>
+
+  <div class="project z">
+       <div class="icon-detail"><img class="icon-det" src="images/AI1.png" alt=""></div>
+       <div class="title-detail">IAEA</div>
+       <div class="descrip">Introduction to AI and its applications in education assessment</div>
+  </div>
+
+  <div class="project">
+       <div class="icon-detail"><img  class="icon-det" src="images/AI2.png" alt=""></div>
+       <div class="title-detail">PAGSS</div>
+       <div class="descrip">AI-powered automated grading and scoring systems</div>
+  </div>
+
+  <div class="project">
+       <div class="icon-detail"><img  class="icon-det" src="images/AI3.png" alt=""></div>
+       <div class="title-detail">AEG</div>
+       <div class="descrip">NLP (Natural Language Processing) and its role in automated essay grading</div>
+  </div>
+
+  <div class="project">
+       <div class="icon-detail"><img  class="icon-det" src="images/AI4.png" alt=""></div>
+       <div class="title-detail">EPSDEN</div>
+       <div class="descrip">Evaluation of AI-powered assessment systems in different educational contexts</div>
+  </div>
+
+  <div class="project a">
+       <div class="icon-detail"><img  class="icon-det" src="images/AI5.png" alt=""></div>
+       <div class="title-detail">IALSG</div>
+       <div class="descrip">The impact of AI-powered assessment on learning outcomes and student performance</div>
+  </div>
+</div>
+  <div class="bntsumitJoin">
+       <button class="parcipateBnt" type="submit">contribute to this best topic</button>
+  </div>
+  <div class="LinksumitJoin">
+       <a class="parcipate" href="">See The Whole Program</a>
+  </div>
+
+<!-------------------------------------------------- PLACE FOR Speaker by Dynamic page------------------------------------------------------- -->
+<div class="title-Main2">Featured Speakers</div>
+<div class="dashProjectMain2"></div>
+<div class="speakers">
+       <div class="speaker-person1">
+              <img class="imgP1" src="" alt="">
+              <div class="flex1">
+                     <div class="T1P1 text1"></div>
+                     <div class="T2P1 text2"></div>
```

Comment:
- [Optional] Dear @massazoum, You did great addressing the suggested changes so far. As a frontend engineer, we are expected to deliver something identical to the design, and I belive your design seems to have more margin compared with the original design. Pleas consider making some arrangements. 
 
![image](https://user-images.githubusercontent.com/69077061/227313559-ca5b0c24-18bd-40f2-92fa-82ad058e3f51.png)